assets/src/dashboard/app/views/myStories/index.js

```diff
@@ -80,15 +80,18 @@ function MyStories() {
   const [status, setStatus] = useState(STORY_STATUSES[0].value);
   const [typeaheadValue, setTypeaheadValue] = useState('');
   const [viewStyle, setViewStyle] = useState(VIEW_STYLE.GRID);
+  const [currentStorySort, setCurrentStorySort] = useState(
+    STORY_SORT_OPTIONS.LAST_MODIFIED
+  );
   const { pageSize } = usePagePreviewSize();
   const {
     actions: { fetchStories },
     state: { stories },
   } = useContext(ApiContext);
 
   useEffect(() => {
-    fetchStories({ status });
-  }, [fetchStories, status]);
+    fetchStories({ status, orderby: currentStorySort });
+  }, [fetchStories, status, currentStorySort]);
```

Comment:
I like how this just re-fetches data when that story sort updates rather than a bunch of callbacks.